documentation/modules/adding-source-provider.adoc

```diff
@@ -207,4 +207,12 @@ ifdef::vmware,rhv,ostack,cnv,cnv2[]
 The *Provider details* page opens.
 .. Click the *Edit* icon under *External UI web link*.
 .. Enter the link and click *Save*.
++
+[NOTE]
+====
+If you do not enter a link, {project-short} attempts to calculate the correct link.
+
+* If {project-short} succeeds, the link is visible on the page. 
```

Comment:
```suggestion
* If {project-short} succeeds, the field's hyperlink will point to the calculated link". 
. 
```

Something like the above...I suggest it because it's a hyperlink, need to mouse over the link in order to see the target url...

Reply:
Fixed.